CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## [16.2.0] - 2020-06-30
+## [16.2.0] - 2020-07-01
```

Comment:
So, you are releasing 16.2.0 but snapshot is 16.1.1 ?

Reply:
Yeah that will be changed when I perform the release